db/pre-re/item_db.txt

```diff
@@ -2316,7 +2316,7 @@
 5133,Sheep_Hat,Sheep Hat,4,20,,150,,1,,0,0x00008110,7,2,256,,0,0,205,{ bonus bShortWeaponDamageReturn,5; },{},{}
 5134,Pumpkin_Hat,Pumpkin-Head,4,20,,200,,2,,0,0xFFFFFFFF,7,2,256,,0,1,206,{ bonus2 bSubRace,RC_Demon,5; },{},{}
 5135,Cyclops_Visor,Cyclop's Eye,4,0,,200,,0,,0,0xFFFFFFFE,7,2,512,,75,0,207,{ bonus bMaxSP,50; },{},{}
-5136,Santa's_Hat_,Antonio's Santa Hat,4,20,,100,,3,,0,0xFFFFFFFF,7,2,256,,0,1,20,{},{},{}
+5136,Santa's_Hat_,Antonio's Santa Hat,4,20,,100,,2,,0,0xFFFFFFFF,7,2,256,,0,1,20,{},{},{}
```

Comment:
kRO description (renewal) give 4 def, iRO give 0 def, aegis (renewal) 0
Source?

Reply:
It's 2 according to Aegis pre-renewal
(and 4 according to my Aegis renewal, not 0 ._.)